Dockerfile

```diff
@@ -15,8 +15,13 @@ COPY ./docker/php/php.ini /usr/local/etc/php/php.ini
 
 RUN curl -sS https://getcomposer.org/installer | php -- --install-dir=/usr/local/bin/ --filename=composer
 
-RUN pecl install xdebug \
- && docker-php-ext-enable xdebug
+RUN pecl channel-update pecl.php.net && \
```

Comment:
no tak to rovnou ten docker image zvedni na 8.3, žádná servisa na 7.x neběží

Reply:
Zvednu teda image na 8.3 a constraint v composeru na `>=8.2`, jako to mají servisy, co tu knihovnu používají.